src/ui/layout/building-adaptive-apps.md

```diff
@@ -1191,8 +1191,8 @@ Here are some resources to get you started:
 
 [Build high quality apps (Android)]: {{site.android-dev}}/quality
 [Human interface guidelines (Apple)]: {{site.apple-dev}}/design/human-interface-guidelines/
-[Material design for large screens]: {{site.material}}/blog/material-design-for-large-screens
+[Material design for large screens]: {{site.material2}}/blog/material-design-for-large-screens
 [Machine sizes and breakpoints (Microsoft)]: https://docs.microsoft.com/en-us/windows/uwp/design/layout/screen-sizes-and-breakpoints-for-responsive-design
-[Material guidelines on responsive UI layout]: {{site.material}}/archive/guidelines/layout/responsive-ui.html
+[Material guidelines on responsive UI layout]: {{site.material2}}/archive/guidelines/layout/responsive-ui.html
```

Comment:
There are better resources now under https://m3.material.io/foundations/layout/applying-layout and https://m3.material.io/foundations/layout/canonical-layouts/overview